### PR TITLE
yaz: build with GnuTLS to allow https transport

### PR DESCRIPTION
**Pull request by adamdickmeiss**
Depend on package gnutls for this.

Build depend on tclsh for git checkout builds.

### CODE REVIEW

File: Formula/yaz.rb [hunk 1]
```diff
@@ -34,17 +34,20 @@ class Yaz < Formula
     depends_on "autoconf" => :build
     depends_on "automake" => :build
     depends_on "libtool" => :build
+    depends_on "tclsh" => :build
```
Comment (carlocab) [inline Formula/yaz.rb:37]: `tclsh` is not a formula, so this is not a valid dependency. Do you mean `tcl-tk`?

**Review by carlocab — COMMENTED** [review]
Thanks for your PR, @adamdickmeiss.

Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings.

For new formulae:

> At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.

For existing formulae:

> The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.

Refer to the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit) for more details. Also, when making further changes to your pull request, use the following guidelines to make sure that @BrewTestBot can merge your commits:

> - One formula per commit; one commit per formula.
> - Keep merge commits out of the pull request.

Also, please bump the `revision` (either add `revision 1` or increment the `revision` right after `license`) so that users who have this installed get the updated version of the formula when they do `brew upgrade`. Thanks!

File: Formula/yaz.rb (hunk 1 above)
Comment (adamdickmeiss) [reply Formula/yaz.rb:37]: Just needs tclsh. If tcl-tk has that, fine.

**Comment by adamdickmeiss** [comment]
Closing this as the commit message was not good (and keeping tclsh out of this).

**Pull request closed without merging**